website/src/css/general.css

```diff
@@ -177,3 +200,22 @@ ul ul ul .table-of-contents__link {
   border-left: 1px solid #03a9f4;
   font-weight: 500;
 }
+
+.blog-post-list {
+  display: grid;
+  grid-template-columns: repeat(3, minmax(0, 1fr));
+  grid-template-rows: mosiac;
```

Comment:
this isn't a valid value, yeah?  i assume it's not doing anything

Reply:
lol, i was trying to get this masonry layout working for a while https://www.smashingmagazine.com/native-css-masonry-layout-css-grid/ but couldn't (not a lot of browser support)

then i tried it again later, and clearly forgot that it masonry - and forgot how to spell mosaic haha

removed